kore/test/Test/Kore/Step/Function/Integration.hs

```diff
@@ -488,15 +496,15 @@ test_functionIntegration =
                             )
                             (definitionEvaluation
                                 [ axiom
-                                    (Mock.f (mkElemVar Mock.y))
+                                    (Mock.f (mkElemVar Mock.yRule))
                                     Mock.a
                                     makeTruePredicate
                                 ]
                             )
                         )
                     ]
                 )
-                (Mock.f (mkElemVar Mock.x))
+                (Mock.f (mkElemVar Mock.xConfig))
```

Comment:
Here and in other tests for the function evaluator/simplification: I think the distinction between the variables in the patterns and the ones in the equations is confusing. To me, it implies that the config/rule distinction matters for equation application when it shouldn't.